### ISSUE TITLE
different pages_per_range show's same pg_size for bin index

### ISSUE THREAD

**Issue by BaiShaoqi** [greenplum-db/gpdb#9053]
### Greenplum version or build
master branch commit 32cc07cc77a07be30da0ea9d2a6a2d3a498339f9

### Expected behavior
the column `pg_size_pretty` should be different among `brin_index_64`, `brin_index_128`,`brin_index_256`, `brin_index_512`
### Actual behavior
```
SELECT relname, pg_size_pretty(pg_relation_size(oid)) from pg_class  where relname='t1' or relname like 'brin_index_%';
    relname     | pg_size_pretty
----------------+----------------
 t1             | 3438 MB
 brin_index_64  | 384 kB
 brin_index_128 | 384 kB
 brin_index_256 | 384 kB
 brin_index_512 | 384 kB
(5 rows)
```

### Step to reproduce the behavior
```psql
create table t1 as select generate_series(1,100000000) as c1;
create index brin_index_64 on t1 using brin(c1) with (pages_per_range=64);
create index brin_index_128 on t1 using brin(c1) with (pages_per_range=128);
create index brin_index_256 on t1 using brin(c1) with (pages_per_range=256);
create index brin_index_512 on t1 using brin(c1) with (pages_per_range=512);
```

**Comment by d** [comment]
Aha! I took a quick look at this, turns out Greenplum works really well with BRIN: each BRIN index we've created has _exactly_ 3 pages (one meta, one revmap, one regular) on each of the 3 segments and the master node. Why? Because Greenplum uses a large (32kb) page size. That means one regular data page in a BRIN index can hold 16 times more ranges than a normal (8k page sized) Postgres cluster.

 This is a curious case of "it looks strange" but it turned out OK.
